docs/source/developer_guide/plugins/contributing.rst

```diff
@@ -52,11 +59,13 @@ Interfaces
 Pluginloader (aiida/common/pluginloader.py)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The pluginloader relies on reentry (PYPI package) to find and load entry points. reentry has been added to setup_requires for aiida in order to enable scanning for existing plugins when aiida is installed. If for some reason reentry should be uninstalled, the plugin system will fall back on pkg_resources from setuptools, which is slower.
+The pluginloader relies on `reentry`_ (PYPI package) to find and load entry points. ``reentry`` has been added to setup_requires for AiiDA in order to enable scanning for existing plugins when AiiDA is installed. If for some reason reentry is uninstalled or is not found, the plugin system will fall back on ``pkg_resources`` from setuptools, which is slower.
```

Comment:
In the link it's not possible http://docutils.sourceforge.net/FAQ.html#is-nested-inline-markup-possible but I'll fix the rest